packages/playground/blueprints/src/lib/steps/define-wp-config-consts.ts

```diff
@@ -8,42 +8,46 @@ export interface DefineWpConfigConstsStep {
 	step: 'defineWpConfigConsts';
 	/** The constants to define */
 	consts: Record<string, unknown>;
+	/** Whether to virtualize the wp-config.php and playground-consts.json files, by default false */
+	virtualize: boolean;
 }
 
 /**
  * Sets site URL of the WordPress installation.
  *
  * @param playground The playground client.
- * @param wpConfigConst
+ * @param wpConfigConst An object containing the constants to be defined and the optional virtual file system configuration file path.
+ * @returns Returns the virtual file system configuration file path.
  */
 export const defineWpConfigConsts: StepHandler<
 	DefineWpConfigConstsStep
-> = async (playground, { consts }) => {
+> = async (playground, { consts, virtualize = false }) => {
 	const documentRoot = await playground.documentRoot;
-	await updateFile(
-		playground,
-		`${documentRoot}/playground-consts.json`,
-		(contents) =>
-			JSON.stringify({
-				...JSON.parse(contents || '{}'),
-				...consts,
-			})
+	let basePath = documentRoot;
+	if (virtualize) {
+		playground.mkdir(VFS_CONFIG_FILE_BASENAME);
+		basePath = VFS_CONFIG_FILE_BASENAME;
```

Comment:
Yay! By the way, is there any reason not to set the php.ini option here? Otherwise the executor needs to do some reasoning based on the steps.

Reply:
@sejas ping ^ :-)

Reply:
@adamziel ,
Ha!, thanks for the suggestion. Really smart!
https://github.com/WordPress/wordpress-playground/pull/415/commits/41b332fc3a2ae103e04b1e8811190c93cee603c2